docs/source/conf.py

```diff
@@ -67,34 +66,39 @@
 # -- Options for HTML output ----------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.
-html_theme = 'jupyter_alabaster_theme'
+html_theme = 'alabaster'
 
-#html_theme_options = {}
-#html_theme_path = []
-#html_title = None
-#html_short_title = None
-#html_logo = None
-#html_favicon = None
+html_logo = '_static/images/logo/favicon.ico'
+html_favicon = '_static/images/logo/logo.png'
```

Comment:
I think these two lines are exactly the wrong way around :) and what causes the logo in the RH corner be "very low res".